README.md

```diff
@@ -36,6 +36,59 @@ The client keys used with Parse are no longer necessary with parse-server.  If y
 * restAPIKey
 * dotNetKey
 
+#### OAuth Support
+
+parse-server supports 3rd party authentication with
+
+* twitter
+* meetup
+* linkedin
+* google
+* instagram
+* facebook
+
+You don't need to explicitely to anything but for Twitter
```

Comment:
If appIds are not set, there is no validation as some providers don't provide an app verification mechanic.